src/standalone/Vegetation/pmodel.jl

```diff
@@ -911,44 +920,53 @@ function update_photosynthesis!(p, Y, model::PModel, canopy)
     Aj = @. lazy(c3_light_assimilation(J, ci, Γstar))
 
     # dark respiration
+    # Here we make an assumption about how to relate Rd25 to Vcmax25_opt
+    # To extend to C4, defined `compute_dark_respiration_pmodel() which dispatches off of the is_c3 field
+    # This function below would become c3_dark_respiration_pmodel
     @. Rd =
         constants.fC3 *
-        (
-            inst_temp_scaling_rd(
-                T_canopy,
-                constants.To,
-                constants.aRd,
-                constants.bRd,
-            ) / inst_temp_scaling(
-                T_canopy,
-                T_canopy,
-                constants.To,
-                constants.Ha_Vcmax,
-                constants.Hd_Vcmax,
-                constants.aS_Vcmax,
-                constants.bS_Vcmax,
-                constants.R,
-            )
-        ) *
-        Vcmax
+        p.canopy.photosynthesis.OptVars.Vcmax25_opt *
```

Comment:
I changed this because it seemed like we were multiplying Vcmax25_opt by the scaling to get Vcmax, and then dividing by the scaling here. It felt clearer to me to just use Vcmax25_opt.